### ISSUE TITLE
Custom tickets button not showing 

### ISSUE THREAD

**Issue by sanmont3drepo** [3drepo/3drepo.io#3975]
### Description
The custom tickets button doesnt show up if you just login

- This is happening in: staging
- This requires a hotfix to production: no 

### Steps to replicate
- go to https://staging.dev.3drepo.io/ (you should have to login, if you are already logged in , logout and refresh the page)
- select a teamspace
- select a federation

### Current Behaviour
-  The custom tickets button is not showing (only after you refresh the page)

### Expected Behaviour
- Custom tickets button should be showing

**Comment by Amantini1997** [comment]
Merge into #3887
